action.yml

```
@@ -55,16 +57,54 @@ runs:
         git commit -m "$TITLE"
         git status
         git push --set-upstream origin "$BRANCH"
+
+    # Trigger CI manually as the pull-request created using the GitHub API later will NOT trigger any new
+    # workflows e.g. ci.yml as a measure to protect against infinite loops
+    # https://github.com/peter-evans/create-pull-request/issues/48#issuecomment-537478081
+    - name: Trigger CI
+      id: trigger-ci
+      uses: silverstripe/gha-trigger-ci@v1
+      with:
+        branch: ${{ inputs.branch }}
+        validate_branch: false
+
+    - name: Update description
+      id: update-description
+      shell: bash
+      env:
+        BRANCH: ${{ inputs.branch }}
+        DESCRIPTION: ${{ inputs.description }}
+        GITHUB_REPOSITORY: ${{ github.repository }}
+      run: |
+        # Escape / to %2F e.g. pulls/1.9/lorem => pulls%2F1.9%2Florem
+        ESCAPED_BRANCH=${BRANCH//\//%2F}
+        UPDATED_DESRCRIPTION=${DESCRIPTION//\"/\\\"}
```

```suggestion
        # Escape double quotes '"' => '\"'
        UPDATED_DESRCRIPTION=${DESCRIPTION//\"/\\\"}
```
Otherwise it looks like the comment above is for this as well, which it doesn't - and someone looking at this might not know to look for the comment two steps above to see what this _is_ doing.

---

Updated